stack-generator.js

```diff
@@ -22,6 +22,9 @@
 
             var curr = arguments.callee;
             while (curr && stack.length < maxStackSize) {
+                if (!curr['arguments']) {
```

Comment:
Nitpick: Would it be a bit simpler to add this as part of the `while()` condition?